integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils.py

```diff
@@ -213,7 +213,47 @@ def _compile_module(module, exported_names, backend_info, artifacts_dir):
 
   _compile_module = _setup_mlir_crash_reproducer(_compile_module, artifacts_dir,
                                                  backend_info.backend_id)
-  return _compile_module(module, exported_names, backend_info, artifacts_dir)
+  return _compile_module(module, backend_info, exported_names, artifacts_dir)
+
+
+def _incrementally_compile_tf_signature_def_saved_model(
```

Comment:
nit: (and I probably should have said this in a previous review). But "incrementally" is kind of confusing here. compile_tf_signature_def_saved_model_with_intermediate_artifacts is a bit better.

But for now, just keep it as "incrementally" for consistency with the other function.

Reply:
SGTM, though I'll need to come up with something that I can still get return arguments out of in 80 characters lol.